docker-compose.yml

```diff
@@ -4,6 +4,9 @@
 
 version: '3.1'
 
+include:
+  - "mock-ica/docker-compose-mock-ica.yml"
```

Comment:
If we split the mocks into various categories - we can include them from the master docker compose - but also spin them indvidually as is needed